crates/maple_config/src/lib.rs

```diff
@@ -14,10 +14,9 @@ fn load_config(
     specified_config_file: Option<PathBuf>,
 ) -> (Config, PathBuf, Option<toml::de::Error>) {
     let config_file = specified_config_file.unwrap_or_else(|| {
-        // Linux: ~/.config/vimclap/config.toml
-        // macOS: ~/Library/Application\ Support/org.vim.Vim-Clap/config.toml
+        // Linuxi & macOS: ~/.config/vimclap/config.toml
         // Windows: ~\AppData\Roaming\Vim\Vim Clap\config\config.toml
-        let config_file_path = Dirs::project().config_dir().join("config.toml");
+        let config_file_path = Dirs::config_dir().join("config.toml");
```

Comment:
Can we keep the support of ` ~/Library/Application\ Support/org.vim.Vim-Clap/config.toml` on macOS? This will ensure compatibility for existing macOS users, preventing any complaints about their current configuration file no longer working.

Reply:
I can add the logic to look on both places.